doc/source/cluster/ray-client.rst

```diff
@@ -152,4 +152,15 @@ If you are using the ``nginx-ingress-controller``, you may be able to resolve th
         nginx.ingress.kubernetes.io/server-snippet: |
           underscores_in_headers on;
           ignore_invalid_headers on;
+
+Ray client logs
+~~~~~~~~~~~~~~~
+
+Ray client logs can be found at ``/tmp/ray/session_latest/logs`` on the head node.
    
+Uploads to the head node
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running ``ray.connect()``, the Ray client will upload the project directory on the laptop to ``/tmp/ray/session_latest/runtime_resources/_ray_pkg_<hash of directory contents>``. If a ``working_dir`` is specified in the runtime env, this ``working_dir`` on the laptop will be uploaded instead of the project directory. In other words, the ``working_dir`` / runtime env mechanism is always used under the hood, but if the ``working_dir`` is not specified, then ``working_dir`` is set to the project directory by default.
```

Comment:
```suggestion
When running ``ray.init()``, the Ray client will upload the project directory on the laptop to ``/tmp/ray/session_latest/runtime_resources/_ray_pkg_<hash of directory contents>``. If a ``working_dir`` is specified in the runtime env, this ``working_dir`` on the laptop will be uploaded instead of the project directory. In other words, the ``working_dir`` / runtime env mechanism is always used under the hood, but if the ``working_dir`` is not specified, then ``working_dir`` is set to the project directory by default.
```